src/components/tab-title/tab-title.tsx

```diff
@@ -355,12 +449,25 @@ export class TabTitle implements InteractiveComponent {
     this.controls = tabIds[titleIds.indexOf(this.el.id)] || null;
   }
 
+  //--------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  //--------------------------------------------------------------------------
+
+  closeClickHandler = (event: MouseEvent): void => {
+    event.stopPropagation();
```

Comment:
Do we need to stop propagation here? Seems like we should allow clicks to bubble here.